ckanext/canada/helpers.py

```diff
@@ -774,6 +777,48 @@ def date_field(field, pkg):
     return pkg.get(field, None)
 
 
+def registry_network_access():
+    """
+    Only allow requests from GOC network to access
+    user account registration view
+    """
+    if not has_request_context():
```

Comment:
@RabiaSajjad Yeah, so looks like the `identify` only happens in `identify_user` in the Flask app class. So there is always be a context. So you can get rid of this `if not has_request_context` and the `has_request_context` import.